src/main/resources/db/changelog/changes/04-create-user-roles-table.yaml

```
@@ -0,0 +1,37 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-user-roles-table
+      author: trokhim
+      preConditions:
+        - onFail: MARK_RAN
+        - not:
+            tableExists:
+              tableName: user_roles
```

have you run you app? 
```suggestion
              tableName: users_roles
```